### ISSUE TITLE
Replace removed ProgressBar with more stable Indicator

### ISSUE THREAD

**Issue by SecUpwN** [SecUpwN/Android-IMSI-Catcher-Detector#678]
After removing the glitchy ProgressBar with #673, I consider it important to now re-add a more stable visual Indicator. I've digged through many projects and collected some ideas. Feel free to add yours!

* [SmoothProgressBar](https://github.com/castorflex/SmoothProgressBar) (different designs, my current favourite)

![SmoothProgressBar](https://github.com/castorflex/SmoothProgressBar/raw/master/screenshots/SPB_sample.gif)

---

* [MaterialLoadingProgressBar](https://github.com/lsjwzh/MaterialLoadingProgressBar)

![MaterialLoadingProgressBar](https://github.com/lsjwzh/MaterialLoadingProgressBar/raw/master/screen.gif)

---

* [android-PullRefreshLayout](https://github.com/baoyongzhang/android-PullRefreshLayout)

![android-PullRefreshLayout](https://raw.githubusercontent.com/baoyongzhang/android-PullRefreshLayout/master/demo.gif)

---

* [SwipyRefreshLayout](https://github.com/OrangeGangsters/SwipyRefreshLayout)

![SwipyRefreshLayout](https://github.com/OrangeGangsters/SwipyRefreshLayout/raw/master/app/src/main/res/raw/github_gif.gif)

---

* [Android-MaterialRefreshLayout](https://github.com/android-cjj/Android-MaterialRefreshLayout)

![Android-MaterialRefreshLayout](https://camo.githubusercontent.com/32ea1ba23dda29aea87572ee247c12a98586eaa7/687474703a2f2f7777772e61706b6275732e636f6d2f646174612f6174746163686d656e742f666f72756d2f3230313530392f31312f3039353835396b703239376d6a6d6a3270687035706d2e6a7067)

**Comment by larsgrefer** [comment]
We already have a SwipeRefreshLayout: http://developer.android.com/reference/android/support/v4/widget/SwipeRefreshLayout.html
So there is no need for an extra dependency i think

**Comment by SecUpwN** [comment]
>So there is no need for an extra dependency i think

I agree. But what needs to be added now in order to make the SwipeRefreshLayout show?

**Comment by SecUpwN** [comment]
>We already have a SwipeRefreshLayout

@larsgrefer, are projects like [Ultra-Pull-To-Refresh](https://github.com/liaohuqiu/android-Ultra-Pull-To-Refresh) and [MaterialProgressBar](https://github.com/DreaminginCodeZH/MaterialProgressBar) of use for us?

**Comment by larsgrefer** [comment]
Everything we would need from Ultra-Pull-To-Refresh is the material style we already have in the support-v4 library.

Our progressbars are already material styled by the appcompat-v7 library

So imho we don't need any extra libraries at the moment

**Comment by SecUpwN** [comment]
>So imho we don't need any extra libraries at the moment

Hm.. so why don't we have a progress indicator showing then? :P

**Comment by larsgrefer** [comment]
Be patient

**Comment by SecUpwN** [comment]
>Be patient

Sorry, I will.

**Comment by SecUpwN** [comment]
Seems like everyone is very busy these days. @CellularPrivacy/design please help to fix this. Reading https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/issues/678#issuecomment-163327400 and https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/issues/678#issuecomment-167161732, it appears that we do not need to add more depencies, but just **enhance the existing pull-to-refresh to work across our whole app.** Comments are very welcome.
